### DISCUSSION TITLE
AppleTv: how to select and send a specific App, located on the AppleTV-Homescreen and assign it to a button?

### DISCUSSION THREAD

**Discussion by andreasbuff** [PRProd/HA-Firemote#254] [General]
Hello, Thank you very much for this very great and genious Integration!  It is very appreciated, especially, as this not only covers Android, or FireTV, but also other brands such as Apple! Great!

Now I have a simple question: If I want to select, for example, the App, placed on the very first row on the top left, of the AppleTV-Homescreen?  Can I do this and if yes, how? 
The app is a local TV-App, provided from a local Internet-Provider.  It would be great, if we could assign this app to the "app_launch_1:" key, or "app_launch_2"

Thanks for your reply

**Comment by PRProd** [comment]
Greetings @andreasbuff!

The Firemote card connects directly to the [Apple TV integration](https://www.home-assistant.io/integrations/apple_tv) which makes this process a lot easier than you might think.  

Thankfully, It doesn't matter how you've arranged your apps on the Apple TV Homescreen, Firemote simply uses the integration to tell your Apple TV device to launch an app by its name.

All you need to do is open a new [App Shortcut Request](https://github.com/PRProd/HA-Firemote/issues/new?assignees=PRProd&labels=New+App+Shortcut%2Cenhancement&projects=&template=App_Shortcut_Request.yaml&title=%5BNew+App+Shortcut%5D%3A+) issue.  Since it sounds like your app might not be available in my region, I would recommend that you also [follow these instructions](https://github.com/PRProd/HA-Firemote/wiki/How-Can-I-Help%3F#for-apple-tv-devices) to fill out the **Application Info** field.

I personally process all of the requests, and make them available in an upcoming Firemote release.
